DTM/Icescar/map.json

```diff
@@ -108,8 +108,8 @@
 		}
 	],
 	"itemremove": [
-		"stone sword", "leather chestplate", "leather helmet", "iron axe", "iron sword", "golden axe", "diamond sword", "arrow", "iron boots", "golden chestplate", 
-		"iron chestplate", "diamond chestplate", "diamond boots", "stone pickaxe", "iron pickaxe", "diamond pickaxe", "diamond block", "emerald block"
+		"stone sword", "stone pickaxe", "splash potion", "bread", "gold ingot",
+		"leather helmet", "leather chestplate"
```

Comment:
I think there might be that many item removes because it's possible to get better armor on this map.